### ISSUE TITLE
Ambient Animations - Search Bar

### ISSUE THREAD

**Issue by Longshot133** [zen-mod/ZEN#688]
**Is your enhancement related to a problem?**
During Ops I may decide to use ambient animations on AI to simulate them discussing topics or simple for worldbuilding. However It can get tedious to scroll through the somewhat large list of Animations over and over. I typically know what animation I wish to have however they are not alphabetical so i must search. (I tend to forget the exact location).

**Solution you'd like:**
I would much appreciate a simple search bar or function to the module to allow faster and easier location of this.

**Alternatives you've considered:**
Possibly Filters Such as; "Medical", "Combat" and such.

**Additional context:**
N/A

**Comment by Longshot133** [comment]
And me being the idiot i am realised 10 seconds after posting i can type while in the menu to search...
(tho its kinda hidden so idk XD)

**Comment by TheRadiumDude** [comment]
Yeah, you can press a letter to seek the next item in the list starting with that letter. I'm not sure if a search bar is very useful, considering there's only a small amount of ambient animations comparatively IIRC.
